main.nf

```diff
@@ -389,35 +389,26 @@ if(!params.accessionList) {
             .ifEmpty { exit 1, "Cannot find any reads matching: ${params.reads}\nNB: Path needs to be enclosed in quotes!\nNB: Path requires at least one * wildcard!\nIf this is single-end data, please specify --single_end on the command line." }
             .into { raw_reads_inspect ; raw_reads_fastqc; raw_reads_trimgalore }
     }
-
 raw_reads_inspect.view()
 }
 
-if(params.accessionList) {
-    Channel
-        .fromPath( params.accessionList )
-        .splitText()
-        .map{ it.trim() }
-        .dump(tag: 'AccessionList content')
-        .ifEmpty { exit 1, "Accession list file not found in the location defined by --accessionList. Is the file path correct?" }
-        .into { accessionIDs_inspect ; accessionIDs }
-
-    accessionIDs_inspect.view()
-}
+/*
+ *  Create channel for the sra-tools accession list
+ */
 
+// Input list .csv file of many .csv files when hbadeals is not skipped
+if(params.accessionList) { Channel.fromPath( params.accessionList ).ifEmpty { exit 1, "Input accession list not found at ${params.accessionList}. Is the file path correct?" } }
+if(params.accessionList) { accessionIDs = Channel.fromPath( params.accessionList ).splitText().unique().map{ it -> it.trim() } }
```

Comment:
#### `unique()` 

This is necessary to avoid errors of type (see below) in case the user hasn't provided a list of unique entries.
```console
input file name collision -- There are multiple input files for each of the following file names:
```

#### `map{ it -> it.trim() }` 
This is necessary to avoid errors caused by the truncation of the `.command.sh` from new line characters in case the user hasn't provided a properly trimmed list.